Add TWIM 2025-05-09 (#2729)
matrix-org/matrix.org: content/blog/2025/05/2025-05-09-twim.md

+++
date = "2025-05-09T17:00:00Z"
title = "This Week in Matrix 2025-05-09"
path = "/blog/2025/05/09/this-week-in-matrix-2025-05-09"

[taxonomies]
author = ["MTRNord"]
category = ["This Week in Matrix"]
+++

> ## Matrix Live

Reviewer: ```suggestion
## Matrix Live S10E23 – Making P2P Matrix more reliable with ReP2P Matrix
```
I'ma start doing this

{{youtube_player(video_id="0dRWKVSAh1U")}}

## Dept of elections 🗳️

[Robin Riley (m.org)](https://matrix.to/#/@robinriley:matrix.org) reports

> It's [Governing Board](https://matrix.org/foundation/governing-board/) election season and nominations opened last weekend! This year we're electing representatives for Gold, Silver, Individual, and Associate Members. Learn more on the [2025 election page](https://matrix.org/foundation/governing-board-elections/2025/) and consider nominating yourself! You can also drop by the [Office of the Governing Board](https://matrix.to/#/#governing-board-office:matrix.org) to talk to current representatives and community members.

## Dept of Servers 🏢

> ### Matrix Authentication Service (MAS) 0.16.0 ([website](https://github.com/element-hq/matrix-authentication-service/))

Reviewer: ```suggestion
### Matrix Authentication Service ([website](https://github.com/element-hq/matrix-authentication-service/))
```
we should discourage people from submitting improper titles

> MAS (Matrix Authentication Service) is an OAuth 2.0 and OpenID Provider server for Matrix.

Reviewer: I [suggest](https://github.com/matrix-org/twim-config/issues/43)
```suggestion
Matrix Authentication Service (MAS) is an OAuth 2.0 and OpenID Provider server developed by Element.
```
to match synapse

[Andrew Morgan (anoa) {he/him}](https://matrix.to/#/@andrewm:element.io) says

> [MAS v0.16.0](https://github.com/element-hq/matrix-authentication-service/releases/tag/v0.16.0) was released this week with a new migration tool! It's still called `syn2mas`, but it's faster, more accurate and... written in Rust. 🦀 Running it against matrix.org showed users being migrated at a rate of 60k/s!
>
> More details around the new tool, and migrating your homeserver to next-generation auth, is [available in the MAS documentation](https://element-hq.github.io/matrix-authentication-service/setup/migration.html). Please give it a try and [send feedback](https://github.com/element-hq/matrix-authentication-service/issues/)!

> 

Reviewer: ```suggestion

<!-- more -->

```
